ops/lib.rs

```diff
@@ -34,8 +34,25 @@ fn core_import() -> TokenStream2 {
   }
 }
 
+#[derive(Debug)]
+struct MacroArgs {
+  is_unstable: bool,
+}
+
+impl syn::parse::Parse for MacroArgs {
+  fn parse(input: syn::parse::ParseStream) -> syn::Result<Self> {
+    let vars =
+      syn::punctuated::Punctuated::<Ident, syn::Token![,]>::parse_terminated(
+        input,
+      )?;
+    let is_unstable = vars.iter().any(|v| v == "unstable");
```

Comment:
Can we cause an error for anything that is not `unstable`?

Reply:
```rs
error: Ops expect #[op] or #[op(unstable)]
  --> core/examples/hello_world.rs:16:1
   |
16 | #[op(wrong)]
   | ^^^^^^^^^^^^
```